src/OpenLoco/src/Ui/ViewportInteraction.cpp

```diff
@@ -918,9 +918,21 @@ namespace OpenLoco::Ui::ViewportInteraction
                     {
                         interactionsToExclude |= InteractionItemFlags::building | InteractionItemFlags::headquarterBuilding | InteractionItemFlags::industry;
                     }
+                    if (vp->hasFlags(ViewportFlags::seeThroughTrees))
+                    {
+                        interactionsToExclude |= InteractionItemFlags::tree;
+                    }
                     if (vp->hasFlags(ViewportFlags::seeThroughScenery))
                     {
-                        interactionsToExclude |= InteractionItemFlags::tree | InteractionItemFlags::wall;
+                        interactionsToExclude |= InteractionItemFlags::wall;
+                    }
+                    if (vp->hasFlags(ViewportFlags::seeThroughTracks))
+                    {
+                        interactionsToExclude |= InteractionItemFlags::track | InteractionItemFlags::trackExtra | InteractionItemFlags::station | InteractionItemFlags::signal;
```

Comment:
Station unfortunately covers airports, docks, road stations and track stations. I'm not sure this is what you would want to disable.

Reply:
Ahh, that's unfortunate. You're right, of course. I was confused with the `InteractionItem` that does have separate entries for airports and docks, but `InteractionItemFlags` does not have them. I'll remove the station bit.